Client/YooChooseNotifier.php

```diff
@@ -136,7 +134,16 @@ protected function notify(array $events)
 
         $response = $this->guzzle->post(
             $this->getNotificationEndpoint(),
-            array( 'json' => array( 'transaction' => null, 'events' => $events ) )
+            array(
+                'json' => array(
+                    'transaction' => null,
+                    'events' => $events
+                ),
+                'auth' => array(
```

Comment:
The value expected by YC is `customer_id/license_key`. I don't think this will produce the expected header.